pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -82,6 +82,7 @@
 , bpftools
 , libbpf
 
+, withAcl ? true
```

Comment:
Can you update the `systemdMinimal` definition (in `pkgs/top-level/all-packages.nix`) to also set these two new options to false if possible?

Reply:
Note we want to enable idn2 again for `systemdStage1Network`.